docs/code-reviews.md

```diff
@@ -41,15 +41,15 @@ and state what you are looking for in the description.
 
 After you make requested changes in response to code review feedback, please re-request reviews from the reviewers to notify them that the work is ready to be reviewed again.
 
-Advantages of code review
-
-catch and prevent bugs
-consistent code
-find shared code
-share knowledge
-
-Challenges
-it can take long
-who to ask
-how do you know when is "enough" review
-what should i be reviewing
+#### Advantages of code review
```

Comment:
Nit: maybe just h2?
```suggestion
## Advantages of code review
```

Reply:
Good call. Mistakenly thought this was a subheading of the above